compiler-rt/lib/sanitizer_common/CMakeLists.txt

```diff
@@ -239,6 +243,11 @@ append_list_if(SANITIZER_LIMIT_FRAME_SIZE -Wframe-larger-than=570
 append_list_if(COMPILER_RT_HAS_WGLOBAL_CONSTRUCTORS_FLAG -Wglobal-constructors
                SANITIZER_CFLAGS)
 
+# Suppress -Watomic-alignment warnings by not treating them as errors
+if (CMAKE_SYSTEM_NAME MATCHES "AIX")
+  list(APPEND SANITIZER_CFLAGS "-Wno-error=atomic-alignment")
+endif()
+
```

Comment:
This is ineffective for handling usage of `compiler-rt/lib/sanitizer_common/sanitizer_atomic_clang.h` from builds of files outside `sanitizer_common`.